packages/functional-tests/tests/settings/recoveryPhone.spec.ts

```diff
@@ -18,9 +18,13 @@ import { getCode } from 'fxa-settings/src/lib/totp';
 
 test.describe('severity-1 #smoke', () => {
   test.describe('recovery phone', () => {
-    test.beforeEach(async ({ pages: { configPage } }) => {
+    test.beforeEach(async ({ pages: { configPage } }, { project }) => {
       // Ensure that the feature flag is enabled
       const config = await configPage.getConfig();
+      test.skip(
+        project.name !== 'local',
+        'recovery phone tests currently only work in local/ci environment'
+      );
```

Comment:
Are we intending to have them work in stage & prod again? if yes, then the fixme annotation seems more appropriate here.